src/mame/machine/tosh1000_bram.h

```diff
@@ -0,0 +1,57 @@
+// license:BSD-3-Clause
+// copyright-holders:Sergey Svishchev
+/***************************************************************************
+
+	Toshiba 1000 Backup RAM
+
+	Simulation of interface provided by 80C50 keyboard controller.
+
+***************************************************************************/
+
+#ifndef __TOSH1000_BRAM_H__
+#define __TOSH1000_BRAM_H__
```

Comment:
Please don't use `__` in names - all such names are reserved for the implementation.